MAUI/Popup/popup-positioning.md

```diff
@@ -264,4 +264,300 @@ public class ViewModel : INotifyPropertyChanged
+                       ShowCloseButton="True">
+        <sfPopup:SfPopup.ContentTemplate>
+          <DataTemplate>
+            <Label Text="A pop-up is a graphical user interface display area that suddenly appears in the foreground of the visual interface. Pop-up can be initiated by single or double tap or can simply be timed to occur. A pop-up window should be smaller than the background window or interface; otherwise, its a replacement interface."                               
+                   TextColor="Black" 
+                   FontSize="14" 
+                   FontFamily="Roboto" 
+                   Padding="0,0,0,24" />
+          </DataTemplate>
+        </sfPopup:SfPopup.ContentTemplate>
+      </sfPopup:SfPopup>
+    </Grid>
+  </ContentPage.Content> 
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="C#" hl_lines="17" %} 
+namespace PopupMauiPositioning;
+using Syncfusion.Maui.DataGrid;
+using Syncfusion.Maui.Popup;
+
+public partial class MainPage : ContentPage
+{
+  SfDataGrid dataGrid;
+  ViewModel viewModel;
+  SfPopup sfPopup;
+
+  public MainPage()
+  {
+    InitializeComponent();
+    dataGrid = new SfDataGrid();
+    viewModel = new ViewModel();
+    dataGrid.ItemsSource = viewModel.OrdersInfo;
+    dataGrid.CellTapped += OnDataGridCellTapped;
+    dataGrid.ColumnWidthMode = ColumnWidthMode.Fill;
+
+    sfPopup = new SfPopup();
+    sfPopup.HeaderTitle = "Popup";
+    sfPopup.ShowCloseButton = true;
+    sfPopup.AutoSizeMode = PopupAutoSizeMode.Height;
+    sfPopup.ContentTemplate = new DataTemplate(() =>
+    {
+        var label = new Label();
+        label.Text = "A pop-up is a graphical user interface display area that suddenly appears in the foreground of the visual interface. Pop-up can be initiated by single or double tap or can simply be timed to occur. A pop-up window should be smaller than the background window or interface; otherwise, its a replacement interface.";
+        label.TextColor = Colors.Black;
+        label.FontSize = 14;
+        label.FontFamily = "Roboto";
+        label.Padding = new Thickness(0, 0, 0, 24);
+        return label;
+    });
+    this.Content = dataGrid;
+  }  
+}
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void OnDataGridCellTapped(object sender, Syncfusion.Maui.DataGrid.DataGridCellTappedEventArgs e)
+{
+  sfPopup.Show();
+}
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with DataGrid ](Images/popup-positioning/show-maui-popup-when-grid-cell-tap.png)
+
+Download the entire source code from GitHub [here](https://github.com/SyncfusionExamples/how-to-show-a-.net-maui-popup-when-tap-.net-maui-datagrid-cell).
+
+### Open SfPopup in ItemTapped event of SfListView
+
+The `SfPopup` allows opening it in the [ItemTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTapped) event of the `SfListView`.

```

Comment:
The `SfPopup` allows opening it in the [ItemTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTapped) event of the `SfListView.`